tests/js/spec/actionCreators/committers.spec.jsx

```diff
@@ -0,0 +1,98 @@
+import {getCommitters} from 'app/actionCreators/committers';
+import CommitterActions from 'app/actions/committerActions';
+import CommitterStore, {getCommitterStoreKey} from 'app/stores/committerStore';
+
+describe('CommitterActionCreator', function() {
+  const organization = TestStubs.Organization();
+  const project = TestStubs.Project();
+  const event = TestStubs.Event();
+
+  const storeKey = getCommitterStoreKey(organization.slug, project.slug, event.id);
+  const endpoint = `/projects/${organization.slug}/${project.slug}/events/${event.id}/committers/`;
+
+  const api = new MockApiClient();
+  const mockData = {
+    committers: [
+      {
+        author: TestStubs.CommitAuthor(),
+        commits: [TestStubs.Commit()],
+      },
+    ],
+  };
+  let mockResponse;
+
+  beforeEach(() => {
+    MockApiClient.clearMockResponses();
+    mockResponse = MockApiClient.addMockResponse({
+      url: endpoint,
+      body: mockData,
+    });
+
+    CommitterStore.init();
+
+    jest.restoreAllMocks();
+    jest.spyOn(CommitterActions, 'load');
+    jest.spyOn(CommitterActions, 'loadSuccess');
+
+    /**
+     * XXX(leedongwei): We would want to ensure that Store methods are not
+     * called to be 100% sure that the short-circuit is happening correctly.
+     *
+     * However, it seems like we cannot attach a listener to the method
+     * See: https://github.com/reflux/refluxjs/issues/139#issuecomment-64495623
+     */
+    // jest.spyOn(CommitterStore, 'load');
```

Comment:
should we keep this code commented out? I'd remove them

Reply:
This test is for a hack around a quirk of Reflux.

The test code that was commented out is the "correct" way to test this hack, but doesn't work due to some other quirks. So I ended up implementing a workaround. 

I feel that it is useful to keep it around for add additional context that the "correct" way was considered and implemented.